scipy/optimize/_minimize.py

```diff
@@ -206,10 +206,12 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
 
         where ``intermediate_result`` is a keyword parameter containing an
         `OptimizeResult` with attributes ``x`` and ``fun``, the present values
-        of the parameter vector and objective function. Note that the name
-        of the parameter must be ``intermediate_result`` for the callback
-        to be passed an `OptimizeResult`. These methods will also terminate if
-        the callback raises ``StopIteration``.
+        of the parameter vector and objective function. If `jac` or `hess` are
+        specified as `Callable` they will also be present in 
```

Comment:
```suggestion
        callable, they will also be present in 
```
Is `Callable` a class? I've only seen it used as English.

Also, often these functions estimate the gradient/Hessian numerically if they are not provided as callables. Are these estimates not available?

Reply:
`Callable` should be at leat something one can use as a type hint, so I thought I use it as such.

These estimates might be available, but I checked how the final results are usually returned and there for instance the hessian is only provided, if a function to evaluate the hessian analytically is given.

Reply:
> should be at least something one can use as a type hint, so I thought I use it as such.

If you can find precedent elsewhere in `scipy.optimize`, sounds good. Since it's not a parameter, it would be need to be in double backticks unless Sphinx recognizes it. Otherwise, please change it.

Reply:
I have changed it back to just callable.